sdk/resourcemanager/Azure.ResourceManager.Core/src/Generated/ManagementGroup/ManagementGroupInfo.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.ResourceManager.Core
+{
+    /// <summary>
+    /// A class representing a ManagementGroup along with the instance operations that can be performed on it.
+    /// </summary>
+    public class ManagementGroupInfo : ManagementGroupOperations
```

Comment:
Went with ManagementGroupInfo / ManagementGroup instead of ManagementGroup / ManagementGroupExpanded?